src/main/java/org/apache/commons/io/output/WriterOutputStream.java

```diff
@@ -448,6 +448,7 @@ public void write(final byte[] b) throws IOException {
      */
     @Override
     public void write(final byte[] b, int off, int len) throws IOException {
+        IOUtils.checkFromIndexSize(b, off, len);
```

Comment:
Javadoc `@throws IndexOutOfBoundsException ...` is missing.

Reply:
Fixed in https://github.com/apache/commons-io/pull/790/commits/8d520f72c3698440e2275b860a991a41757ad2fb